tests/set-namespace/empty-input/simple/.expected/results.yaml

```diff
@@ -1,4 +1,12 @@
-name: set-namespace
+apiVersion: kpt.dev/v1alpha2
+kind: FunctionResultList
+metadata:
+    name: fnresults
+exitCode: 1
 items:
-- message: 'failed to configure function: input namespace cannot be empty'
-  severity: error
+    - image: gcr.io/kpt-fn/set-namespace:unstable
+      stderr: '[error] /// : failed to configure function: function config must be a ConfigMap or SetNamespaceConfig'
```

Comment:
what's going on here?

Reply:
As mentioned in my earlier comment above.
https://github.com/GoogleContainerTools/kpt/issues/2109#issuecomment-849010352 is causing this issue. I will create a PR for it in kyaml this week. It will land and be release after some time. We will bump the kyaml dependencies in functions after that. I don't think we want to block this PR because of this minor issue.